documentation/docs/25-build-and-deploy/70-adapter-cloudflare-workers.md

```diff
@@ -108,7 +108,9 @@ declare global {
 export {};
 ```
 
-> `platform.env` is only available in your build, you can use [wrangler](https://developers.cloudflare.com/workers/cli-wrangler) to test it locally.
+### Testing Locally
+
+`platform.env` is only available in the final build and not in dev mode. For testing the build we can use [wrangler](https://developers.cloudflare.com/workers/cli-wrangler). Once you have built your site, run `wrangler dev`. Ensure you have your [bindings](https://developers.cloudflare.com/workers/wrangler/configuration/#bindings) in your `wrangler.toml`. Wrangler version 3 is recommended.
```

Comment:
This section is a bit different between this adapter and the other one. We should probably make it the same

Reply:
Synced your `we` -> `you` change from the other comment - the other differences are quite deliberate. Wrangler 3 is required for pages but only recommended for workers

Reply:
Ah, okay!